scalableminds/webknossos
# Fix dropdown login padding

This PR adds a margin between the `forgot password` and `Register Now` in the login dropdown and thus ensures some space between them. I also made the position of the dropdown fixed so that it does not scroll away anymore.

### URL of deployed dev instance (used for testing):
- [https://fixdropdownloginissue.webknossos.xyz](https://fixdropdownloginissue.webknossos.xyz)

### Steps to test:
- logout and open the dropdown. Resize the window and try to get the two options `forgot password` and `Register Now` together. There should be at least some space between them. 
- Testing that the dropdown does not scroll away: add some `margin-bottom` to the main container so that it becomes scrollable :)

### Issues:
- fixes #3956

------
